src/extension.ts

```diff
@@ -22,20 +22,20 @@ export function activate(context: vscode.ExtensionContext) {
   vscode.commands.registerTextEditorCommand(
     COMMAND_NAME,
     (editor, edit, args: CommandArgs) => {
-      edit.insert(
-        args.position.translate(0, -args.old_prefix.length),
-        args.new_prefix
-      );
-      edit.insert(
-        args.position.translate(0, args.old_suffix.length),
-        args.new_suffix
-      );
-      edit.delete(new vscode.Range(
-        args.position.translate(0, -args.old_prefix.length),
-        args.position.translate(0, args.old_suffix.length),
-      ));
-      let new_position = args.position.translate(0, -args.old_prefix.length);
-      editor.selection = new vscode.Selection(new_position, new_position);
```

Comment:
I have removed the code that sets the new position and the `editor.selection`, only because from my testing I didn't seem to need it for the cursor to behave as expected.

I'm happy to add this back though and map `editor.selections` to be the new selections based on `new_position`.